compiler/luci/pass/src/FusePreActivationBatchNormPass.test.cpp

```diff
@@ -0,0 +1,264 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "FusePreActivationBatchNormPassInternal.h"
+
+#include <luci/IR/CircleNodes.h>
+
+#include <math.h>
+#include <vector>
+
+#include <gtest/gtest.h>
+
+namespace
+{
+
+/**
+ *  Simple graph for test
+ *
+ *  BEFORE
+ *
+ *          [Conv]  W + bias
+ *            |
+ *          [Add]
```

Comment:
If this is `[Add]`, there should be another input(s) but the diagram doesn't provide them...

Reply:
The pass only requires one CONV, so I made a a test with a single input for simplicity. Another node can be added, but it may not affect the test.

Reply:
Um... does this work also? (sorry, I drawn it in horz direction)
```
[Conv] --- [Add] --- [Mul] --- [Add+Relu] -- [Conv] --- [Add]
[Conv] --/         \----------------------------------/

Reply:
>  I'm curious if the transformations works if there is two Conv and then Add...

Yes, it works. In that case, `beta/gamma` is added to the bias of one Conv. I'll add one more Conv to show the case.